addons/resource.language.en_gb/resources/strings.po

```diff
@@ -16412,7 +16412,7 @@ msgstr ""
 #. Description of setting #14102 Settings -> Video -> Discs -> Blu-ray playback mode
 #: system/settings/settings.xml
 msgctxt "#37031"
-msgid "Specifies how Blu-rays should be opened/played back. Disc menus are not fully supported yet and may cause problems."
+msgid "Specifies how Blu-ray's should be opened/played back. Disc menus are not fully supported yet and may cause problems."
```

Comment:
Although this might be correct (I'm not a native English speaker), "Blu-ray's, CD's, ..." looks somehow strange to me...